<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register Student</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Montserrat ExtraBold</family>
           <pointsize>14</pointsize>
           <weight>10</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 81 14pt &quot;Montserrat ExtraBold&quot;;</string>
         </property>
         <property name="text">
          <string>Register New Student</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 57 10pt &quot;Montserrat Medium&quot;;</string>
           </property>
           <property name="text">
            <string>Student ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditStudentID">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(242, 248, 255);
color: rgb(14, 23, 67);
font: 57 10pt &quot;Montserrat Medium&quot;;
border-radius: 10px;
padding: 5px;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font: 57 10pt &quot;Montserrat Medium&quot;;</string>
           </property>
           <property name="text">
            <string>Full Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditFullName">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(242, 248, 255);
color: rgb(14, 23, 67);
font: 57 10pt &quot;Montserrat Medium&quot;;
border-radius: 10px;
padding: 5px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUpload">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton {
 /* Màu nền mặc định */
	background-color: rgb(254, 212, 1);
    border-radius: 5px;
	color: rgb(255, 123, 0);
	font: 57 10pt &quot;Montserrat Medium&quot;;

}

QPushButton:hover {
background-color: rgb(255, 220, 42);
}

QPushButton:pressed {
    background-color: #F0C800; /* Màu khi nhấn */
}


</string>
         </property>
         <property name="text">
          <string>Upload Photo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCapture">
         <property name="styleSheet">
          <string notr="true">
QPushButton {
 /* Màu nền mặc định */
	background-color: rgb(254, 212, 1);
    border-radius: 5px;
	color: rgb(255, 123, 0);
	font: 57 10pt &quot;Montserrat Medium&quot;;

}

QPushButton:hover {
background-color: rgb(255, 220, 42);
}

QPushButton:pressed {
    background-color: #F0C800; /* Màu khi nhấn */
}


</string>
         </property>
         <property name="text">
          <string>Capture Photo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelPhoto">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 57 10pt &quot;Montserrat Medium&quot;;
border-color: rgb(255, 234, 17);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Student Photo</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="styleSheet">
      <string notr="true">font: 57 10pt &quot;Montserrat Medium&quot;;</string>
     </property>
     <property name="text">
      <string>Ready to register</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRegister">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #35B6E8; /* Màu nền mặc định */
	color: rgb(255, 255, 255);
    border-radius: 5px;
	font: 57 10pt &quot;Montserrat Medium&quot;;
    padding: 5px;

}

QPushButton:hover {
    background-color: #A3E3FC; /* Màu khi di chuột */

}

QPushButton:pressed {
    background-color: #35B6E8; /* Màu khi nhấn */
}

QPushButton:focus {
    border: 5px solid #A3E3FC; /* Viền khi nút được focus */
}
</string>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #35B6E8; /* Màu nền mặc định */
	color: rgb(255, 255, 255);
    border-radius: 5px;
	font: 57 10pt &quot;Montserrat Medium&quot;;
    padding: 5px;

}

QPushButton:hover {
    background-color: #A3E3FC; /* Màu khi di chuột */

}

QPushButton:pressed {
    background-color: #35B6E8; /* Màu khi nhấn */
}

QPushButton:focus {
    border: 5px solid #A3E3FC; /* Viền khi nút được focus */
}
</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
